setup.cfg

```diff
@@ -17,4 +17,4 @@ ignore_missing_imports = True
 ignore_errors = True
 
 [mypy-azure_functions_worker._thirdparty.typing_inspect]
-ignore_errors = True
\ No newline at end of file
+ignore_errors = True
```

Comment:
Do we still need this file?

Reply:
for now yes since we have all mypy configs in there. But eventually it should be removed